apps/web/src/views/Ifos/hooks/v1/useGetPublicIfoData.ts

```diff
@@ -47,7 +47,7 @@ const useGetPublicIfoData = (ifo: Ifo): PublicIfoData => {
           } as const),
       )
 
-      const client = publicClient({ chainId: ChainId.BSC })
+      const client = getViemClients({ chainId: ChainId.BSC })
```

Comment:
Why not this to match other changes?
```suggestion
    const client = viemClients[ChainId.BSC]
```